circuits/cpp/src/aztec3/utils/types/circuit_types.hpp

```diff
@@ -12,78 +13,63 @@ namespace aztec3::utils::types {
+    using uint16 = plonk::stdlib::uint16<Builder>;
+    using uint32 = plonk::stdlib::uint32<Builder>;
+    using uint64 = plonk::stdlib::uint64<Builder>;
 
     // Types related to the bn254 curve
-    using fr = stdlib::field_t<Builder>;
-    using safe_fr = stdlib::safe_uint_t<Builder>;
-    using address = stdlib::address_t<Builder>;
-    using fq = stdlib::bigfield<Builder, barretenberg::Bn254FqParams>;
+    using fr = plonk::stdlib::field_t<Builder>;
+    using safe_fr = plonk::stdlib::safe_uint_t<Builder>;
+    using address = plonk::stdlib::address_t<Builder>;
+    using fq = plonk::stdlib::bigfield<Builder, barretenberg::Bn254FqParams>;
 
-    using witness = stdlib::witness_t<Builder>;
+    using witness = plonk::stdlib::witness_t<Builder>;
 
     // typedef fq grumpkin_fr;
     // typedef fr grumpkin_fq;
-    using grumpkin_point = stdlib::point<Builder>;  // affine
-    using grumpkin_group = stdlib::group<Builder>;
+    using grumpkin_point = plonk::stdlib::cycle_group<Builder>;  // affine
 
-    using bn254 = stdlib::bn254<Builder>;
+    using bn254 = plonk::stdlib::bn254<Builder>;
     // typedef bn254::g1_ct bn254_point;
-    using bn254_point = stdlib::element<Builder, fq, fr, barretenberg::g1>;  // affine
+    using bn254_point = plonk::stdlib::element<Builder, fq, fr, barretenberg::g1>;  // affine
 
-    using bit_array = stdlib::bit_array<Builder>;
-    using byte_array = stdlib::byte_array<Builder>;
-    using packed_byte_array = stdlib::packed_byte_array<Builder>;
+    using bit_array = plonk::stdlib::bit_array<Builder>;
+    using byte_array = plonk::stdlib::byte_array<Builder>;
+    using packed_byte_array = plonk::stdlib::packed_byte_array<Builder>;
 
-    using schnorr_signature = stdlib::schnorr::signature_bits<Builder>;
-    using ecdsa_signature = stdlib::ecdsa::signature<Builder>;
+    using schnorr_signature = plonk::stdlib::schnorr::signature_bits<Builder>;
+    using ecdsa_signature = plonk::stdlib::ecdsa::signature<Builder>;
 
-    using AggregationObject = stdlib::recursion::aggregation_state<bn254>;
-    using VK = stdlib::recursion::verification_key<bn254>;
+    using AggregationObject = plonk::stdlib::recursion::aggregation_state<bn254>;
+    using VK = plonk::stdlib::recursion::verification_key<bn254>;
     // Notice: no CircuitType for a Proof: we only ever handle native; the verify_proof() function swallows the
     // 'circuit-type-ness' of the proof.
 
-    /// TODO: lots of these compress / commit functions aren't actually used: remove them.
-
-    // Define the 'circuit' version of the function `compress`, with the name `compress`:
-    static fr compress(std::vector<fr> const& inputs, const size_t hash_index = 0)
+    static crypto::GeneratorContext<curve::Grumpkin> get_generator_context(const size_t hash_index)
     {
-        return plonk::stdlib::pedersen_commitment<Builder>::compress(inputs, hash_index);
+        crypto::GeneratorContext<curve::Grumpkin> result;
```

Comment:
Here is where we were previously adding an Aztec domain string and using generatorIndexDomain